docker-compose.yml

```diff
@@ -34,6 +37,13 @@ services:
       - redis_data:/redis/data:cached
     restart: on-failure
 
+  kafka:
+    image: apache/kafka:latest
```

Comment:
it's generally recommended to pin infra dependencies to specific versions to ensure runs are deterministicc. let's use the latest one for now, and adjust after we have actual infra